channel/handlers.go

```diff
@@ -4336,3 +4533,210 @@ func (server *Server) playerUseStorage(conn mnet.Client, reader mpacket.Reader)
 		return
 	}
 }
+
+func (server *Server) playerHandleMessenger(conn mnet.Client, reader mpacket.Reader) {
+	plr, err := server.players.getFromConn(conn)
+	if err != nil {
+		return
+	}
+
+	mode := reader.ReadByte()
+
+	p := mpacket.CreateInternal(opcode.ChannelPlayerMessengerEvent)
```

Comment:
Internal packets should exist in: https://github.com/Hucaru/Valhalla/blob/master/internal/packets.go